index.js

```diff
@@ -41,16 +79,11 @@ app.on('ready', () => {
     app.setBadgeCount(unreadCount);
   });
 
-  function getConfig() {
-    let config;
-
-    try {
-      const file = fs.readFileSync(config_path);
-      config = JSON.parse(file);
-    } catch(e) {
-      config = {notificationMode: 'never'};
+  ipcMain.on('showMainWindow', (event) => {
+    if (mainWindow && !mainWindow.isVisible()) {
+      mainWindow.show();
+      event.returnValue = true;
```

Comment:
Renderer process is blocked until return value from Main process. If without return value, Renderer process forever stop.